### ISSUE TITLE
How to reinitialize animationData with new JSON

### ISSUE THREAD

**Issue by suryakanta101** [airbnb/lottie-web#2477]
We have Lottie animation initialize with x.json and after sometime/some event we want to load another animation from y.json in same container.

How this is possible ?

We tried to destroy the 1st animation and change animationData property and lottie.play, but it is not working. New animation not showing

**Comment by bodymovin** [comment]
hi, how are you changing animationData? Are you loading a new instance of the player?

**Comment by suryakanta101** [comment]
Yes , i want to create new instance of animation based on asynchronous response. 

Like example , my Animation Json coming from a ajax call where i need to update the Animation everytime by respective Json data

**Comment by bodymovin** [comment]
yes, that is possible to do, you call your loading function again passing the new data.

**Comment by ajinkyakambe** [comment]
But if I call the loading function again. I get isLoaded: false in the new Lottie object
